app/models/supplemental_file.rb

```diff
@@ -16,12 +16,16 @@ class SupplementalFile < ApplicationRecord
   has_one_attached :file
 
   # TODO: the empty tag should represent a generic supplemental file
-  validates :tags, array_inclusion: ['transcript', 'caption', '', nil]
+  validates :tags, array_inclusion: ['transcript', 'caption', 'machine_generated', '', nil]
```

Comment:
I was bringing it up now to try to avoid a data migration in the future, but a data migration should be fairly simple to do in the future if needed since this is in the database and not fedora.